packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -163,6 +164,12 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Semanti
   /// See [Window.onTextScaleFactorChanged].
   @protected
   void handleTextScaleFactorChanged() { }
+  
+  /// Called when the platform brightness changes.
+  /// 
```

Comment:
The would be a good place to explain where a handler would find the platform brightness value (I assume we want a statement about MediaQueryData here).

Reply:
I reworked the docs to show multiple query options.